sdk/core/Azure.Core/tests/FailedResponseExceptionTests.cs

```diff
@@ -418,7 +456,7 @@ private Stream GetStream(bool isSeekable, string content)
                 false => new UnSeekableStream(_stream)
             };
         }
-        private class UnSeekableStream : MemoryStream
+        private class UnSeekableStream : Stream
```

Comment:
I changed this to not be a MemoryStream in order to test the unbuffered stream path, as this no longer relies on CanSeek but rather on whether or not the stream is a MemoryStream.

Reply:
Curious: would it make sense to have one for each of MemoryStream and Stream to test both paths?

Reply:
Ah, sorry I missed that.  What is the MemoryStream path now?

Reply:
It is handled here - https://github.com/Azure/azure-sdk-for-net/pull/35716/files#diff-09267fe4d81a44f807f352dcef44e16df3c06ad1cfef85be399b4d826a9ebb60R454